### DISCUSSION TITLE
Please delete this Discussion.....

### DISCUSSION THREAD

**Discussion by Phoenxi131178** [zwave-js/zwave-js#5332] [Ideas] [answered]
Is it possible to include Fibaro CO Detector to zwave-js please.
It is not in the database and i can´t connected it to Home Assistant with Zwave-js.

**Comment by AlCalzone** [comment, marked as answer]
Please fill out the corresponding issue template, so we have the information we need:
https://github.com/zwave-js/node-zwave-js/issues/new?assignees=&labels=config+%E2%9A%99&template=device_config.yml&title=Missing+device+configuration%3A+%3Center+manufacturer+and+device+name%3E

**Comment by thecobra666** [comment]
I'm curious, could you link the product page for this device? I'm only able to find the CO detector but not the CO2 detector.

**Reply by Phoenxi131178** [reply]
I´m sorry my bad. You are right. I also found out that I bought the homekit version.
I will close the discussion and will try it with the ZWave Version.
